serverless/aws/examples/src/main/scala/sttp/tapir/serverless/aws/examples/TerraformConfigExample.scala

```diff
@@ -0,0 +1,34 @@
+package sttp.tapir.serverless.aws.examples
+
+import io.circe.Printer
+import io.circe.syntax._
+import sttp.tapir.serverless.aws.examples.LambdaApiExample.helloEndpoint
+import sttp.tapir.serverless.aws.terraform.AwsTerraformEncoders._
+import sttp.tapir.serverless.aws.terraform.{AwsTerraformApiGateway, AwsTerraformInterpreter, AwsTerraformOptions, S3Source}
+
+import java.nio.charset.StandardCharsets.UTF_8
+import java.nio.file.{Files, Paths}
+
+/** Before running the actual example we need to interpret our api as Terraform resources */
+object TerraformConfigExample extends App {
+
+  val jarPath = Paths.get("serverless/aws/examples/target/jvm-2.13/tapir-aws-examples.jar").toAbsolutePath.toString
```

Comment:
can we maybe pass in the "project home" directory as an env variable?

Reply:
I can access `System.getProperty("user.dir")` which gives me the path to tapir root directory, that's not what we need. Or do you thought about some other system property / env variable here?

Reply:
Well I guess this depends a lot on how you run this example. Does this work from intellij or from sbt console?

Reply:
Both, `Paths.get(...)` starts from project root directory so it works regardless of where the project is located